spk/prowlarr/src/wizard/upgrade_uifile

```diff
@@ -1,6 +1,6 @@
 [{
     "step_title": "Updating Prowlarr",
     "items": [{
-        "desc": "Keep Prowlarr up-to-date by using Prowlarr's built-in updater.<br>Navigate to System>Updates in the Prowlarr UI.<br/><br/><b style=\"color: red\">Due to an error in Prowlarr, the automatic update in service start does not work, and you have to start the update manually.</b>"
+        "desc": "Keep Prowlarr up-to-date by using Prowlarr's built-in updater.<br>Navigate to System>Updates in the Prowlarr UI.<br/><br/><b style=\"color: red\">Due to an error in Prowlarr, the automatic update in service start does not work, and you have to start the update manually.</b><a href="https://github.com/Prowlarr/Prowlarr/issues/1199">Issue #1199</a>"
```

Comment:
lint failed

`spk/prowlarr/src/wizard/upgrade_uifile: line 4, col 300, found: 'INVALID' - expected: 'EOF', '}', ':', ',', ']'.`

Reply:
Yeah I know, not sure how to include link in upgrade_uifile, probably just lack of backslash